cli.js

```diff
@@ -1647,8 +1647,9 @@ require('yargs')
   })
 
   .command({
-    command: 'merge master',
-    desc: 'Pull Avo master branch into your current branch',
+    command: 'merge main',
+    aliases: ['merge master'],
```

Comment:
Do we need this for `pull` as well?

Reply:
Yes and no. This is the only place where the _command_ included `master` but we probably want to have `avo pull master` and `avo checkout master` gracefully fall back to `main`. I'll look into it!

Reply:
Yeh, it could just be a soft replace behind the scenes. Then again, one might be sensitive to that being aliased. Maybe the most reasonable thing would be to produce a warning when trying to use `master` and explain that the terminology has changed?

Reply:
I added a check for `master` in the checkout function (which happens to be also called during `avo pull`) about it being deprecated and `main` being the new thing.